apps/stage-web/src/components/Layouts/InteractiveArea.vue

```diff
@@ -134,6 +126,11 @@ watch(showMicrophoneSelect, async (value) => {
   }
 })
 
+onMounted(() => {
+  loadWhisper()
```

Comment:
```suggestion
  // loadWhisper()
```

Let's forget this for now.